src/routes/InventoryTable.js

```diff
@@ -188,7 +183,11 @@ const Inventory = ({
                 <Grid gutter="md">
                     <GridItem span={12}>
                         {
-                            InvCmp && <InvCmp
+                            <AsyncInventory
```

Comment:
The AsyncInventory has providers for RBAC, redux and react-router, could that cause issues when using it here?

Reply:
Could we use just the `RenderWrapper` instead here?

Reply:
On contrary, it should help the inventory to show proper screens when user has no access.